### DISCUSSION TITLE
basic-extra-meal/annotation

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#283] [Announcements]
# basic-extra-meal/annotation

本文深入探讨了Java注解的概念、分类及其在实际项目中的应用。通过详细的示例和解释，帮助读者更好地理解和掌握Java注解技术，学会如何自定义注解以及在实际开发中灵活运用，提升代码的可读性和可维护性。

https://javabetter.cn/basic-extra-meal/annotation.html

**Comment by liweiook** [comment]
“现在是不是豁然开朗了？”我问三妹，看到三妹点了点头。这要是能豁然开朗只能说代码天生圣体，反射都没学到就理解了

**Comment by xiaolixaioyu** [comment]
有点超纲

**Comment by BOUNDLESS-558** [comment]
注解还是要配合反射一起使用才能够有效果，单独一个注解在代码中提供不了任何实际功能。

**Comment by Happy-frog-one** [comment]
这怕是顺序不对吧   我还是多少有点基础的  跟着顺序下来  突然上注解是不是超纲了

**Comment by lumole** [comment]
看不懂，可能三妹是天才吧
